packages/react/select/src/Select.stories.tsx

```diff
@@ -457,6 +457,75 @@ export const WithinForm = () => {
   );
 };
 
+export const RequiredWithinForm = () => {
+  const [data, setData] = React.useState({});
+
+  function handleChange(event: React.FormEvent<HTMLFormElement>) {
+    const formData = new FormData(event.currentTarget);
+    setData(Object.fromEntries((formData as any).entries()));
+  }
+
+  return (
+    <form
+      style={{ padding: 50 }}
+      onSubmit={(event) => {
+        handleChange(event);
+        event.preventDefault();
+      }}
+      onChange={handleChange}
+    >
+      <Label style={{ display: 'block' }}>
+        Name
+        <input name="name" autoComplete="name" style={{ display: 'block' }} />
+      </Label>
+      <br />
+      <Label style={{ display: 'block' }}>
+        Country
+        <Select.Root required name="country" autoComplete="country" defaultValue="">
+          <Select.Trigger className={triggerClass()}>
+            <Select.Value />
+            <Select.Icon />
+          </Select.Trigger>
+          <Select.Portal>
+            <Select.Content className={contentClass()}>
+              <Select.Viewport className={viewportClass()}>
+                <Select.Item className={itemClass()} value="">
```

Comment:
Not sure if it's a bug, but I have to add this item with `value=""`, because when I tried not putting a `defaultValue`, it selects the first item when submitting the form.

Reply:
![image](https://user-images.githubusercontent.com/63862061/182829081-5fabbac9-f651-4754-b0c2-ce453b7b6210.png)

Reply:
I believe this was fixed already a while ago, could you try removing and double-checking?

Reply:
Did you manage to double-check this?

Reply:
Hi, sorry I missed this comment.  I will do it later after workhours.

Reply:
Update on this @benoitgrelard. This is still happening 
![image](https://user-images.githubusercontent.com/63862061/195101923-d51197b0-8302-4000-a16b-d79394099e27.png)

Reply:
Yeah that makes sense because the native select underneath has a value by default even when no default value is given: https://codesandbox.io/s/native-select-required-7kb1g0?file=/src/App.js

Reply:
repro: https://stackblitz.com/edit/react-ts-tgl1rd?file=App.tsx

Reply:
Oh I see, it's potentially an issue that it doesn't match because it prevents `required` to work.
For example if someone doesn't provide a `defaultValue` or `value` and uses `placeholder` on `Value`, it looks like the `Select` is waiting for the user to pick a value and therefore should stop form submitting if `required`, yet it wouldn't.

Reply:
Follow-up: https://github.com/radix-ui/primitives/pull/1711